lib/github_notification_monitor.rb

```diff
@@ -148,6 +149,15 @@ def remove_labels(command_value, author, issue)
     end
   end
 
+  def move_issue(command_value, author, issue)
+    match = /(?:([\w-]+)\/)?([\w-]+)/.match(command_value)
+    fq_repo_name = "#{match[1] || issue.organization_name}/#{match[2]}"
```

Comment:
For clarity and future readability it might be useful to use named captures here...

```ruby
match = /(?:(?<org>[\w-]+)\/)?(?<repo>[\w-]+)/.match(command_value)
fq_repo_name = "#{match[:org] || issue.organization_name}/#{match[:repo]}"
```